modules/parquet/src/parquetjs/classes/parquet-cursor.ts

```diff
@@ -0,0 +1,95 @@
+// Forked from https://github.com/kbajalc/parquets under MIT license (Copyright (c) 2017 ironSource Ltd.)
+import {FileMetaData} from '../parquet-thrift';
+import {ParquetEnvelopeReader} from './parquet-envelope-reader';
+import {ParquetSchema} from '../schema/schema';
+import {ParquetRecord} from '../schema/declare';
+import * as Shred from '../schema/shred';
```

Comment:
Unrelated nit: Would be good to import the actual functions instead of * - there are just two of them.

Reply:
Done